### DISCUSSION TITLE
Problem with exporting logs from nginx running on Docker

### DISCUSSION THREAD

**Discussion by dawidfreelancer** [martin-helmich/prometheus-nginxlog-exporter#374] [General]
I have problem trying to mount access.log and error.log from nginx running on Docker. There is also problem with reading these files using `cat` command. These logs files of nginx running on docker seems to be not real files to read but just symlinks that pass logs to docker stdout so we can read these logs by `docker logs nginx-container`.

**Comment by MatveyIvanov** [comment]
I fixed it with
```dockerfile
RUN unlink /var/log/nginx/access.log
RUN unlink /var/log/nginx/error.log
```
in my nginx Dockerfile. This might not work if you have an existing volume for this files. In that case, delete the volume if possible and try again.
